### DISCUSSION TITLE
French (Français)

### DISCUSSION THREAD

**Discussion by ObjectifMoon21** [sovenor/bitcoin-rocks#10] [Translations]
Salut ! I've started translating the 5 json files below in the screenshot into French.
If other people want to translate pages, let me know here so that we don't work twice on the same pages.
See ya!
![Capture d'écran 2024-05-06 235256](https://github.com/sovenor/bitcoin-rocks/assets/166211858/0d4acda3-6567-4c0e-bde9-6e62c91cf75c)

**Comment by sovenor** [comment]
Thanks for your time on this!

**Reply by ObjectifMoon21** [reply]
you're welcome! It's a pleasure to help you in a bitcoin project. 
Thank you for making it available for everyone in open source

**Comment by ObjectifMoon21** [comment]
The following pages are completed now: 

> `404_fr`, `bank-runs_fr`, `bitcoin-vs-cbdc_fr`, `bitcoin-vs-gold_fr`, `common_fr`, `compound-inflation-calculator_fr`, `get-involved_fr`.

**Comment by ObjectifMoon21** [comment]
The page is translated: 

> `index_fr`

**Comment by ObjectifMoon21** [comment]
The following page is translated in French:

> `inflation_fr`

**Comment by ObjectifMoon21** [comment]
Today i've work on typos + translated more pages in fr:

`lightning_fr` `postcard-success_fr` `postcards_fr` `sticker-language-success_fr` `sticker-success_fr` `stickers_fr` `wallets_fr`

**Comment by ObjectifMoon21** [comment]
Hi @sovenor ! 
Do the pages in the files: `business` `nostr` `sticker-files` need to be translated too ?

**Reply by sovenor** [reply]
Yes, they should all be translated. Most of those pages have very few translation strings so it shouldn't take much time per page.

**Reply by sovenor** [reply]
@ObjectifMoon21 on second thought, you can just translate the main sticker files page + the French sticker files page. I don't think there's really a need to do French language translations for non-French sticker pages.

**Reply by ObjectifMoon21** [reply]
yes, it makes sense, that's what I did. 

I'm going to translate all the pages for the bitcoin business kit, and later on we should make the brochures available in French so the plebs can download the files and print their own. Tell me if you need some help for this.
Yesterday, i've submit a request for the translation of the 'bitcoin accepted here' sticker.

**Comment by sovenor** [comment]
Please note I added a new step to the Translations guide that should be applied to each file:

At the top of the file, change the "last-updated" date to today's date in YYYY-MM-DD format. Change the "locale" to to the [2 letter language code](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) that corresponds to your language. You can also optionally add your name in the list of authors.

Apologies for the oversight! Please make this change on all of the json files.

**Reply by ObjectifMoon21** [reply]
I had started the translations with ''locale'' fr, but seeing the other translators who left it as is I wasn't sure it was necessary lol.
I'll update it all next time!

Earlier I left you a comment on nostr about the stickers in other languages, I would like to know if you are going to use another qr code on it to send to the translated page or it's automatic with the user browser language settings or ip location ?

**Reply by sovenor** [reply]
Thanks, that sounds great! The website should automatically direct the user to the proper language based on their browser language. So if someone in France scans the sticker QR code and their browser language is set to French, it will show everything translated into French.

**Comment by ObjectifMoon21** [comment]
Today I worked on typos + more pages translated in fr :
`nostr\index_fr.json` `nostr\what-is-nostr_fr.json` `sticker-files\french\index_fr.json` `sticker-files\index_fr.json`

**Comment by ObjectifMoon21** [comment]
@sovenor I saw these 2 index files are in a english folders. Do you gonna make other folders for french or i just translate the file in the english folder ?
![Capture d'écran 2024-05-12 133857](https://github.com/sovenor/bitcoin-rocks/assets/166211858/0c0d2d6e-8522-48c7-8f90-e2a9bf9fcf05)

**Reply by sovenor** [reply]
@ObjectifMoon21 English is the only available option for business stickers and business kits right now, so please just translate the files in those English folders.

In the future it would be great to be able to offer the business resources in other languages, but it is a low priority at the moment.

**Comment by ObjectifMoon21** [comment]
The following pages are translated in french :
`business\accounting_fr.json` `business\faq_fr.json` `business\guide_fr.json`

**Comment by ObjectifMoon21** [comment]
C'est génial le site est maintenant disponible en Français ! Si vous ne voyez pas l'option en Français, effacez le cache de votre navigateur et rafraichissez la page.

Vous êtes le bienvenue, si vous voulez aider à la relecture des pages ou pour signaler une erreur de traduction ou autres, n'hésitez pas a laisser un commentaire ici.

I'll check out the website for bugs, proofread + typos and is just missing a few translated pages for the business section.
I'll probably send back a new pull request in a few days.

**Comment by ObjectifMoon21** [comment]
Salut @sovenor  ! 

I've shown the website in French to some friends, and some of them think that the part about inflation sounds like a conspiracy theory against governments and banks. 
I realize I need to find new friends ^^ but in France, people are very cautious of fake news, and check if the informations on the net is carefully sourced. 

Then guess what, i found sources explaining the inflation is the result of the increase in the money supply, on French government websites and even the IMF international money fund website .

I want to know if i can add some short quotes to the inflation page with a note for the source like this for exemple: 
'L'inflation est causée par une création monétaire excessive, plus précisément d'une croissance de la masse monétaire trop forte par rapport à la croissance de la production.' (source gouv Français : vie-publique.fr).

**Reply by sovenor** [reply]
Great idea! Yes, feel free to add that and submit a PR. 

Is the reason they think it sounds like a conspiracy because they believe the source of inflation to be something other than the increase in the money supply? Just trying to understand this more so I can make edits to the text if necessary.

**Reply by ObjectifMoon21** [reply]
yes, some people think that governments and central banks can't print money from thin air, and that new money necessarily comes from new wealth. 

When i ask them where inflation comes from then, they think it's a problem of supply and demand shock, people's overconsumption that pushes prices up. I said all these are the causes of inflation but the primary source is the massive increase in the money supply. 

I've tried to explain to them overconsumption can cause temporary shortages and inflated prices but in long term that should create value and new wealth for everyone, if that was true and well distributed we should see higher salaries and deflation on products like in the golden ages of the post-war periods and other times, but now the total opposite is happening.

**Comment by sovenor** [comment]
@ObjectifMoon21 I noticed an error in my English json file for the home page. I had "home_link_title_art_1" and "home_link_title_art_2" with the same duplicate headlines by mistake.

Please update "home_link_title_art_2" in index_fr.json with the translated version of "Send a Bitcoin Postcard to someone you know"

Thank you!

**Reply by ObjectifMoon21** [reply]
it's done! I'll proofread the french version again soon and send a new PR. 

Btw, i've help someone to start the spanish translation, and I've found another person on nostr for the dutch version. We are pushing ahead ! More languages will be available very soon!

**Reply by sovenor** [reply]
Amazing, you are adding huge value! Thank you for all your work thus far.

**Comment by sovenor** [comment]
Hi, please note that there are 4 new translation strings that are now in "common_fr.json"

"common_self_custody": "SELF-CUSTODY",
"common_not_your_keys": "NOT YOUR KEYS",
"common_cold_wallet": "COLD WALLET",
"common_hot_wallet": "HOT WALLET"

You should see these at the end of the file when you pull the latest changes. Thanks!

**Reply by ObjectifMoon21** [reply]
Hi!
Done, PR is sent: fr translation common_fr.json + phrasing & typos

**Reply by ObjectifMoon21** [reply]
I think i never pull changes from your repo before, between my translations. Maybe that's why there were conflicts on my previous PR.

**Reply by sovenor** [reply]
Ah yes that would cause a conflict! Glad to have found the source of the issue.

**Comment by sovenor** [comment]
A new translation string has been added to index_fr.json:

"home_link_title_energy_6"

Please translate when you have a moment :)

**Reply by ObjectifMoon21** [reply]
done !

**Comment by ObjectifMoon21** [comment]
Translation in French of the " DANGER " sticker: 
![sticker-danger-french](https://github.com/user-attachments/assets/b39919e8-963d-4628-a591-38702477c59e)

**Reply by sovenor** [reply]
This is great! I will work on making all of the source files for all of the stickers available here so that this is easier to do in the future. Most of the stickers are Adobe Illustrator (.ai) files.

**Comment by sovenor** [comment]
Two new translation strings have been added to the top of common_fr.json:

"common_footer_mission_1"
"common_footer_mission_2"

Please translate these when you are able. Thank you!

**Reply by ObjectifMoon21** [reply]
Done! PR sent.

**Reply by sovenor** [reply]
Approved and merged, thank you!

**Comment by sovenor** [comment]
New translation strings have been added to inflation_fr.json:

Line 11: "inflation_sign_got_inflation"
Line 12: "inflation_save_in_bitcoin"
Line 42: "inflation_usd_but_why"

There are also 2 new pages that don't necessarily need translation since the signs campaign is USA only:

signs_fr.json
sign-success_fr.json

Thanks for your continued contributions!

**Reply by ObjectifMoon21** [reply]
Hi! It's done !

As there isn't much text in the pages i've translated the whole thing, adding a note that it's only for the u.s. PR is sent ;)

**Reply by sovenor** [reply]
Thanks! It will take me a little more time since the Spanish translations are included in the PR, but I will get to it soon :)

**Comment by ObjectifMoon21** [comment]
hi @sovenor have you planned to add the translation of [bitcoin is green](https://bitcoinis.green/) ?

I think it would be nice to have this page translated as well.

**Reply by sovenor** [reply]
I will have to rearchitect that site to make it happen but I'll add it to my list!

**Comment by sovenor** [comment]
Translation strings have been modified slightly in wallets_fr.json:

Line 11: "two simple questions" is now "these simple questions"
Line 13: "two simple questions" is now "these simple questions"

These new strings have also been added to the same file:

Lines 33 - 39 to add a section about recovery phrases / seed phrases.

Please translate these if you have some spare time. Thank you!

**Reply by ObjectifMoon21** [reply]
Happy to see this new section about seed phrases. It could help newbies! 

Translation is done and PR is sent, in french and spanish.
